### PR TITLE
Alert + Card Branch

### PR DESCRIPTION
**Pull request by tejas-muthya37**
this branch includes the landing page, introduction page and the alert and card components.

### CODE REVIEW

**Review by Hnh013 — APPROVED** [review]
Alerts and cards look good and distinct !

**Comment by tejas-muthya37** [comment]
thank you! :)

File: Card/card.css [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 0.4rem;
+    padding: 15px;
+}
+
+.footer-icon-group {
+    display: flex;
+    justify-content: flex-end;
+    gap: 30px;
```
Comment (rahulbakshi4) [inline Card/card.css]: Try to have consistency with the units you use prefer rems

File: Card/card.css [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+        font-size: 1.5rem;
+        margin-top: 0.8rem;
+    }
+
+    .badge {
+        position: absolute;
+        bottom: -1.6rem;
+        left: 0;
+        font-size: 0.7rem;
+        width: 80px;
```
Comment (rahulbakshi4) [inline Card/card.css]: Same here too have a consistent units

**Review by rahulbakshi4 — COMMENTED** [review]
Try to maintain a consistency with the units. Look for units at multiple place and prefer using rem

File: Card/card.css (hunk 2 above)
Comment (tejas-muthya37) [reply Card/card.css]: Thanks Rahul. I've fixed this and deployed it. Can you please approve it if there are no other issues?

**Review by rahulbakshi4 — APPROVED** [review]

**Pull request merged**